correspondence/index.bs

```diff
@@ -46,4 +46,196 @@ Issue: Migrate from `APICorrespondence.md`.
+        <td>[#1327](https://github.com/gpuweb/gpuweb/issues/1327)
+        <td>`min(maxImageDimension2D, maxImageDimensionCube)`
+        <td>min(`Maximum 2D texture width and height`, `Maximum cube map texture width and height`)
+        <td>16384 = `min(D3D12_REQ_TEXTURE2D_U_OR_V_DIMENSION, D3D12_REQ_TEXTURECUBE_DIMENSION)`
+    <tr>
+        <th>`maxTextureDimension3D`
+        <td>[#1327](https://github.com/gpuweb/gpuweb/issues/1327)
+        <td>`maxImageDimension3D`
+        <td>`Maximum 3D texture width, height, and depth`
+        <td>2048 = `D3D12_REQ_TEXTURE3D_U_V_OR_W_DIMENSION`
+    <tr>
+        <th>`maxTextureArrayLayers`
+        <td>[#1327](https://github.com/gpuweb/gpuweb/issues/1327)
+        <td>`maxImageArrayLayers`
+        <td>`Maximum number of layers per 1D texture array, 2D texture array, or 3D texture`
+        <td>2048 = `D3D12_REQ_TEXTURE2D_ARRAY_AXIS_DIMENSION`
+    <tr>
+        <th>`maxBindGroups`
+        <td>[#47](https://github.com/gpuweb/gpuweb/issues/47)
+        <td>`maxBoundDescriptorSets`
+        <td class=issue>It's complicated.
+        <td rowspan=3>*Strategy-dependent.* Allocate the `D3D12_MAX_ROOT_COST`, minus reserved slots, across these three limits.
+    <tr>
+        <th>`maxDynamicUniformBuffersPerPipelineLayout`
+        <td>[#406](https://github.com/gpuweb/gpuweb/issues/406)
+        <td>`maxDescriptorSetUniformBuffersDynamic`
+        <td>When not using argument buffers: `maxUniformBuffersPerShaderStage`
+
+            Issue: When using argument buffers: ?
+    <tr>
+        <th>`maxDynamicStorageBuffersPerPipelineLayout`
+        <td>[#406](https://github.com/gpuweb/gpuweb/issues/406)
+        <td>`maxDescriptorSetStorageBuffersDynamic`
+        <td>When not using argument buffers, `maxStorageBuffersPerShaderStage`
+
+            Issue: When using argument buffers: ?
+    <tr>
+        <th>`maxSamplersPerShaderStage`
+        <td>[#409](https://github.com/gpuweb/gpuweb/issues/409)
+        <td>`maxPerStageDescriptorSamplers`
+        <td>`Maximum number of entries in the sampler state argument table, per graphics or kernel function`
+        <td>`Maximum number of Samplers in all descriptor tables per shader stage`
+    <tr>
+        <th>`maxSampledTexturesPerShaderStage`
+        <td>[#409](https://github.com/gpuweb/gpuweb/issues/409)
+        <td>`maxPerStageDescriptorSampledImages`
+        <td rowspan=2>*Strategy-dependent.* Allocate `Maximum number of entries in the texture argument table, per graphics or kernel function` across these two limits.
+        <td>`Maximum number of Shader Resource Views in all descriptor tables per shader stage`
+    <tr>
+        <th>`maxStorageTexturesPerShaderStage`
+        <td>[#409](https://github.com/gpuweb/gpuweb/issues/409)
+        <td>`maxPerStageDescriptorStorageImages`
+        <td rowspan=2>*Strategy-dependent.* Allocate `Maximum number of Unordered Access Views in all descriptor tables across all stages` across these two limits.
```

Comment:
This rowspan, when paired with two offset rowspans in the Metal column, is a little awkward to visually to parse in the preview. I don't really have a suggestion for resolving it, but felt like it was worth pointing out anyway.

<img width="1459" alt="Screen Shot 2022-06-30 at 3 24 38 PM" src="https://user-images.githubusercontent.com/805273/176788799-6d0fad5d-49d3-4d51-a86e-af6153548a7b.png">

Reply:
An alternative would be to do these as footnotes, no rowspans, put "[1]"/"[2]"/"[3]" in each cell and then put the text at the bottom.